python/ray/workflow/tests/conftest.py

```diff
@@ -5,49 +5,79 @@
 
 from ray.tests.conftest import get_default_fixture_ray_kwargs
 from ray._private.test_utils import simulate_storage
+from ray.cluster_utils import Cluster
 
 # Trigger pytest hook to automatically zip test cluster logs to archive dir on failure
 from ray.tests.conftest import pytest_runtest_makereport  # noqa
 from ray.workflow.tests import utils
 
 
 @contextmanager
-def _workflow_start(storage_url, shared, **kwargs):
+def _workflow_start(storage_url, shared, use_ray_client, **kwargs):
+    assert use_ray_client in {"no_ray_client", "ray_client"}
     init_kwargs = get_default_fixture_ray_kwargs()
     init_kwargs.update(kwargs)
     init_kwargs["storage"] = storage_url
-    if ray.is_initialized():
-        ray.shutdown()
+    ray.shutdown()
+    if use_ray_client:
```

Comment:
`use_ray_client` is a str and it should always be True?

Reply:
good catch